test/integration/framework/apiserver.go

```diff
@@ -58,7 +58,7 @@ func RunControlPlane(t *testing.T) (*rest.Config, StopFunc) {
 	patchCRDConversion(crds, webhookOpts.URL, webhookOpts.CAPEM)
 	// environment variables
 	env := &envtest.Environment{
-		AttachControlPlaneOutput: true,
+		AttachControlPlaneOutput: false,
```

Comment:
I think this can be controlled by an environment variable anyway: https://book.kubebuilder.io/reference/testing/envtest.html#configuring-your-test-control-plane